assets/custom/css/style.css

```diff
@@ -310,7 +309,7 @@ footer .list-inline {
   text-transform: uppercase;
   letter-spacing: 0.04em;
   color: #ffffff;
-  //padding-top: 25px;
+  /* padding-top: 25px; */
```

Comment:
Did you check a page with no almost no content? Does this break the footer?

Reply:
/unitelections and /dues are the shortest pages I can think of and if anything, they're too long.